src/query.c

```diff
@@ -980,8 +1017,18 @@ hy_query_filter(HyQuery q, int keyname, int cmp_type, const char *match)
 	HySack sack = query_sack(q);
```

Comment:
forget the comment. Call `reldeps_from_str` and `query_filter_reldep_in` from here. It was previous suggestion to find reldeps after it;s really needed (after `query.run()`)